packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Messages.java

```diff
@@ -942,6 +942,7 @@ public interface Result<T> {
     /** Failure case callback method for handling errors. */
     void error(@NonNull Throwable error);
   }
+
```

Comment:
Are you using a Java formatter other than the one the repo tool's `format` runs? If the Pigeon version didn't change, I don't see why there would be newlines added in this file.